Demos/Blazorise.Demo/Pages/Dashboard.razor

```diff
@@ -1,5 +1,6 @@
 ﻿@page "/"
 @inject IVersionProvider VersionProvider
+@attribute [RouterTabsPageAttribute( Title = "Dashboard", TabCssClass = "font-weight-bold" )]
```

Comment:
The class names should be CSS provider agnostic. So in this case `"font-weight-bold"` would only work for BS.

Reply:
Uhm, it's whatever the user wants, this is just an example.